### PR TITLE
Update epfl.json

### PR DESCRIPTION
**Pull request by grolimur**
removed country field whose value was wrong

### CODE REVIEW

File: engines_json/epfl.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
```
Comment (egh) [inline engines_json/epfl.json:1]: you need to add a `title`

Comment (grolimur) [reply engines_json/epfl.json:1]: Done. https://github.com/grolimur/zotero-lookup-engines/commit/0b3d12cc50303834151233e65d9ab7b7f66253c0

Comment (grolimur) [reply engines_json/epfl.json:1]: Done.

**Comment by egh** [comment]
Looks good!

**Comment by egh** [comment]
http://egh.github.io/zotero-lookup-engines/epfl.html

**Pull request merged**